.circleci/config.yml

```diff
@@ -21,24 +21,30 @@ workflows:
               only:
                 - main
 
+commands:
+  bootstrap:
+    steps:
+      - checkout
+      - run: 'echo "npmAuthToken: $NPM_TOKEN" >> ${HOME}/.yarnrc.yml'
+      - node/install-packages:
+          pkg-manager: yarn-berry
+          with-cache: true
+          include-branch-in-cache-key: false
+
 jobs:
   build:
     parameters:
       publish:
         type: boolean
         default: false
     docker:
-      - image: circleci/node:12.7
+      - image: cimg/node:18.18.0
     working_directory: ~/project
     steps:
-      - checkout
-      - run: 'echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc'
```

Comment:
Do we not need this anymore? I kinda thought I'd seen the echo >> config thing for yarn as well.

Fwiw I think ...one of our libraries...was putting an `NPM_TOKEN = {env.NPM_TOKEN}` type line in the `.yarnrc.yml` instead of doing it via a CI `echo >>`, and I thought that was nice.

Reply:
I DRY-ed out this config by creating a `bootstrap` command (see above). We could also do the inline config, but that'll override any settings people have globally in their home directories. I kinda prefer leaving that config open, but configuring in CI as-needed.

Reply:
Ah shoot, yeah missed the line above. Sgtm!